omc3/scripts/merge_kmod_results.py

```diff
@@ -120,7 +131,7 @@ def _check_tfs_sanity(data_frame: tfs.TfsDataFrame):
         raise KeyError(msg)
 
     # check that there is no weird additional data
-    too_many_entries = [ip for ip in IPS if len(data_frame[NAME].str.startswith(ip)) > 2]
+    too_many_entries = [ip for ip in IPS if sum(data_frame.index.str.startswith(ip)) > 2]
```

Comment:
Maybe use `.sum()` from pandas directly for speed?

Reply:
![image](https://user-images.githubusercontent.com/26184899/94692157-80dac080-0332-11eb-944d-4ccd6b17c93d.png)

valid point to get it into habit. Otherwise this is like <100 entries here... 

fixed it

Reply:
Still saves about 5 us for 50 elements :p